infrastructure-playbooks/cephadm-adopt.yml

```diff
@@ -145,6 +145,19 @@
           inventory_hostname in groups.get(iscsi_gw_group_name, []) or
           inventory_hostname in groups.get(nfs_group_name, [])
 
+    - name: configure repository for installing cephadm
+      when: containerized_deployment | bool
+      block:
+        - name: validate repository variables
+          import_role:
+            name: ceph-validate
+            tasks_from: check_repository.yml
+
+        - name: configure repository
+          import_role:
+            name: ceph-common
+            tasks_from: "configure_repository.yml"
+
```

Comment:
@clwluvw this makes the playbook fail
by the way, `ceph_origin` `ceph_repository` and `ceph_stable_release` have never been required when `containerized_deployment` is `true`, this could be seen more or less as a breaking change if we decide to make those variables mandatory even for containerized deployment.

Reply:
@guits As we discuss above using repository installation for installing cephadm would be better so here because after this playbook ceph-ansible won't be used I think it's okay to set these variables. BTW is it okay to set `ceph_stable_release` in test group vars or do you have better solutions?

Reply:
IMO, `ceph_stable_release` is a legacy from the time where a single branch of ceph-ansible supported more than one release of ceph

Reply:
@guits So remove it? =))))))) or just simple pass it as an extra vars?

Reply:
I think `ceph_stable_release` should be enforced in the role `ceph-defaults` for stable branches

Reply:
@guits Good idea. I will make a PR for it